web/src/cypress/integration/temporarySchedule.ts

```diff
@@ -137,6 +138,7 @@ function testTemporarySchedule(screen: string): void {
       cy.get('[data-cy="shifts-list"]').should('contain', manualAddUser.name)
       cy.dialogClick('Submit')
       cy.get('[data-cy="no-coverage-checkbox"]')
+        .scrollIntoView()
         .should('be.visible')
```

Comment:
Can we get rid of both of these?

`scrollIntoView` should happen automatically on actions click check/click as well as asserting/waiting for visibility of the input field.

Unless I'm mistaken `cy.get("input[name=allowCovoerageGaps]").check()` should handle retry, waiting, scrolling, and ensuring it's visible.